index.js

```diff
@@ -36,8 +38,26 @@ const STATUS_COLOR = {
   INTERNAL_ERROR: '#EA4335', // red
 };
 
+const getGithubCommit = async (build, octokit) => {
+  const cloudSourceRepo = build.source.repoSource.repoName;
+  const { commitSha } = build.sourceProvenance.resolvedRepoSource;
+
+  // format github_ownerName_repoName
+  const [, githubOwner, githubRepo] = cloudSourceRepo.split('_');
+
+  // get github commit
+  const githubCommit = await octokit.git.getCommit({
+    commit_sha: commitSha,
+    owner: githubOwner,
+    repo: githubRepo,
+  });
+
+  // return github commit
+  return githubCommit;
+};
+
 // createSlackMessage create a message from a build object.
-module.exports.createSlackMessage = (build) => {
+module.exports.createSlackMessage = async (build, octokit) => {
```

Comment:
It's a bit weird to pass `octokit` as an argument. 

2 options I can see: 
- Instead of a simple method, have a class on which octobit is attached
- Pass the `githubCommit` as argument instead, and have the parent function fetch it

WDYT?

Reply:
True, it is weird indeed.

I've went with passing the `githubCommit` as argument instead, and having the parent function fetch it.
However the parent function still takes `octokit` as an argument... is that ok?